packages/contact-center/task/src/CallControl/call-control.presentational.tsx

```diff
@@ -45,50 +46,117 @@ function CallControlPresentational(props: CallControlPresentationalProps) {
+      tooltip: isRecording ? 'Pause Recording' : 'Resume Recording',
+      className: 'call-control-button',
+    },
+    {
+      icon: 'cancel-regular',
+      onClick: endCall,
+      tooltip: 'End call',
+      className: 'call-control-button-cancel',
+    },
+  ];
+
+  if (!currentTask) return null;
+
   return (
     <>
       <audio ref={audioRef} id="remote-audio" autoPlay></audio>
-      {currentTask && (
-        <div className="box">
-          <section className="section-box">
-            <fieldset className="fieldset">
-              <legend className="legend-box">Call Control</legend>
-              <div style={{display: 'flex', flexDirection: 'column', flexGrow: 1}}>
-                <div style={{display: 'flex', gap: '1rem'}}>
-                  <ButtonPill onPress={handletoggleHold} disabled={wrapupRequired} color={isHeld ? 'join' : 'cancel'}>
-                    {isHeld ? 'Resume' : 'Hold'}
-                  </ButtonPill>
-                  <ButtonPill onPress={handletoggleRecording} disabled={wrapupRequired}>
-                    {isRecording ? 'Pause Recording' : 'Resume Recording'}
-                  </ButtonPill>
-                  <ButtonPill onPress={endCall} disabled={wrapupRequired || isHeld}>
-                    End
-                  </ButtonPill>
-                </div>
-                <div style={{display: 'flex', gap: '1rem', marginTop: '1rem'}}>
-                  <select className="select" onChange={handleWrapupChange} disabled={!wrapupRequired}>
-                    <option value="">Select the wrap-up reason</option>
-                    {wrapupCodes.map((wrapup: WrapupCodes) => (
-                      <option key={wrapup.id} value={wrapup.id}>
-                        {wrapup.name}
-                      </option>
-                    ))}
-                  </select>
-                  <ButtonPill onPress={handleWrapupCall} disabled={!wrapupRequired && !selectedWrapupReason}>
-                    Wrap Up
-                  </ButtonPill>
-                </div>
+
+      <div className="call-control-container">
+        {!wrapupRequired && (
+          <div className="button-group">
+            {buttons.map((button, index) => (
+              <TooltipNext
+                key={index}
+                color="primary"
+                delay={[0, 0]}
+                placement="bottom-start"
+                triggerComponent={
+                  <ButtonCircle className={button.className} onPress={button.onClick}>
+                    <Icon className={button.className + '-icon'} name={button.icon} />
+                  </ButtonCircle>
+                }
+                type="description"
+                variant="small"
+                className="tooltip"
+              >
+                <p>{button.tooltip}</p>
+              </TooltipNext>
+            ))}
+          </div>
+        )}
+        {wrapupRequired && (
+          <div className="wrapup-group">
+            <PopoverNext
```

Comment:
Let's also check if the popover style matches the Figma. For instance, the border & padding in the Figma and the vidcast doesn't seem to match